### PR TITLE
cmd/tailscale/cli: fix false positive on cli page

### LINKED ISSUE #7059: Synology 1.36.0 claims outgoing connections when I’ve not enabled them
### What is the issue?

I updated both my real Synology and the Virtual DSM Synology to 1.36.0, and on both of them Tailscale claims “Outgoing access enabled” even though on neither of them I’ve done anything to enable it. If I ssh in, no 100.* addresses work, so I suspect the detection of this state is unreliable.

### Steps to reproduce

Install Tailscale 1.36.0 on a Synology DSM.

### Are there any recent changes that introduced the issue?

_No response_

### OS

Synology

### OS version

Synology DSM 7.1.1-42962 Update 2

### Tailscale version

1.36.0

### Bug report

BUG-58e464b3f7e4c645ab93703678cb504b35e27890719d5038d7186929129b905f-20230125034526Z-371a751eeff7ed68

### PR DESCRIPTION
**Pull request by phirework**
Fixes #7059

### CODE REVIEW

File: cmd/tailscale/cli/web.html [hunk 1]
```diff
@@ -99,13 +99,13 @@ <h3 class="text-3xl font-semibold mb-3">Log in</h3>
 		<a href="#" class="mb-4 link font-medium js-loginButton" target="_blank">Reauthenticate</a>
 	</div>
 	{{ if .IsSynology }}
+		{{ if not .TUNMode }}
```
Comment (bradfitz) [inline cmd/tailscale/cli/web.html:102]: and if it is? where'd that text go?

also, please adjust the git title to say something about Synology and web

File: cmd/tailscale/cli/web.html [hunk 2]
```diff
@@ -99,13 +99,13 @@ <h3 class="text-3xl font-semibold mb-3">Log in</h3>
...
 		<div class="border border-gray-200 bg-orange-0 rounded-lg p-2 pl-3 pr-3 mb-8 width-full text-orange-800">
-			Outgoing access {{ if true }}enabled{{ else }}not configured{{ end }}.
-			<nobr><a href="https://tailscale.com/kb/1152/synology-outbound/"
-				class="font-medium link"
-				target="_blank"
-				rel="noopener noreferrer">Learn more &rarr;</a></nobr>
+			Outgoing traffic of services is not configured.
```
Comment (bradfitz) [inline cmd/tailscale/cli/web.html:104]: Can we do the wording change in its own PR?

I'm not a huge fan of this but it's fine if we're consistent, but https://tailscale.com/kb/1131/synology/#enabling-synology-outbound-connections doesn't match this

File: cmd/tailscale/cli/web.html (hunk 1 above)
Comment (phirework) [reply cmd/tailscale/cli/web.html:102]: Discussed with @mayakacz offline and she suggested removing entirely, because calling it out in an alert box didn't make sense when it's a success case, and leaving it up by itself underneath reauth/advertise links looked (imo) messy and didn't follow a clear content hierarchy 

Screen Shot 2023-01-24 at 17 55 02

Comment (bradfitz) [reply cmd/tailscale/cli/web.html:102]: I want something there for debugging with users.

If nothing's there then we don't know whether they're just using an old version that didn't say anything.

Comment (phirework) [reply cmd/tailscale/cli/web.html:102]: But yes I can leave that to a separate PR

**Review by bradfitz — APPROVED** [review]

**Comment by bradfitz** [comment]
maybe "fix TUN mode display" instead of "fix false positive" ?
